docs/opsgenie.md

```diff
@@ -0,0 +1,22 @@
+# OpsGenie
+
+## Get API key
+At first, customers should set up an integration and get API key:
+1. Go to your Opsgenie and select Teams from menu.
+
+2. Select your team to access your team dashboard.
+
+3. Select Integrations from left navigation.
+
+4. Select Add Integration.
+
+5. Select API Integration.
+
+6. Copy `API Key`.
+
+7. When done with all configurations, select Save Integration to enable the integration.
+
+[Set up an integrated tool for Opsgenie](https://support.atlassian.com/opsgenie/docs/set-up-an-integrated-tool/).
+
+Also there is an interesting mark about Authentication problem: [OpsGenie API (Create alert) Authentication problem](https://community.atlassian.com/t5/Opsgenie-questions/OpsGenie-API-Create-alert-Authentication-problem/qaq-p/1531047).
```

Comment:
Can we add the details (of what needs to be done) in this README instead of making users go to a forum?